js-i2b2/cells/PM/PM_admin_view.js

```diff
@@ -137,11 +137,35 @@ i2b2.PM.model.helpMSGS.PROJECTUSERS = "<form><p>Click on \"User\" in the navigat
 	'<table border="0"><tbody><tr><td valign="middle"><b>User Name:</b></td><td><input type="TEXT" id="pmAdmin-userName" maxlength="50" style="width:250px"/></td></tr>'+
 	'<tr><td></td><td align="right"> <input type="BUTTON" value="Add User to Project" onclick="i2b2.PM.admin.addUserProject();"/> <input type="BUTTON" value="Cancel" onclick="i2b2.PM.view.admin.refreshScreen();"/></td></tr>'+
 	'</table></div></form>';
+i2b2.PM.model.helpMSGS.DBLOOKUP =  "<form><p>Click on \"DBLookup\" in the navigation bar to refresh the list of DB Lookup.<br />Please select a DB Lookup on the left to edit it's properties</p>"+
```

Comment:
I don't understand where the patches to this file came from and what they have to do with CAS.